team.html

```diff
@@ -103,7 +100,7 @@ <h3>Miguel David</h3>
             <h3>Elisete Cruz</h3>
             {% include components/teamhandle.html twitter="elisetedcruz" linkedin="elisetecruz" %}
 
-            <p>Devops Engineer, Tools and Automation at <a href="https://www.euronext.com" target="_blank">Euronext</a>.
+            <p>Release Engineer at <a href="https://www.farfetch.com" target="_blank">Farfetch</a>.
```

Comment:
Some bit not aot all links have `target="_blank"`. Not a big deal though... WDYT?

Reply:
Good spot. Fixed!